h5t_test.go

```diff
@@ -65,48 +68,62 @@ func TestStructDatatype(t *testing.T) {
 
 	// Test that the type can be constructed and that the number of
 	// members is as expected.
+	dtypes := make([]*Datatype, 0)
```

Comment:
we usually spell this out as:
```go
var dtypes []*Datatype
```

Reply:
If it is really pressing, I can obviously fix that one.

Reply:
Please do.